packages/labs/signals/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Signal} from 'signal-polyfill';
+
+export * from 'signal-polyfill';
+export * from './lib/signal-watcher.js';
+export * from './lib/watch.js';
+export * from './lib/html-tag.js';
+
+export const State = Signal.State;
+export const Computed = Signal.Computed;
+
+export const signal = <T>(value: T, options?: Signal.Options<T>) =>
```

Comment:
This is my thinking:

The proposal's API is pretty verbose compared to framework library signal APIs, which are published in modules and don't use namespace objects. The intention is that frameworks will wrap the API with more convenient ones. We don't have a framework, but we can offer a couple of really small convenience methods so that devs can use a single import and a shorter syntax and automatically use the native feature in the future.

Compare:

```ts
import {Signal} from 'signal-polyfill';
import {SignalWatcher} from '@lit-labs/signals';

const count = new Signal.State(0);
```

to:

```ts
import {SignalWatcher, state} from '@lit-labs/signals';

const count = state(0);
```

This might not matter much to someone also using signals outside of Lit and are already importing the polyfill, but it'll be nice for developers only using signals with Lit, as if it were a feature of the "framework".

Reply:
Why not this? `export * from 'signal-polyfill';`

Just `new Signal.State(0)` v. `state(0)` alone doesn't seem worth obfuscating the connection between this package the the proposal.

Reply:
> Why not this? `export * from 'signal-polyfill'`;

Because that exports a namespace object, which is not that idiomatic for modules.

You still have this pattern:

```ts
import {SignalWatcher, Signal} from '@lit-labs/signals';

const count = new Signal.State(0);
```

That might not seem so bad with one signal, but with the `watch()` pattern you'll possibly need to create a lot of computed signals to get targeted updates for derived data:

```ts
import {SignalWatcher, state, computed} from '@lit-labs/signals';

export class MyElement extends SignalWatcher(LitElement) {
  #count = state(0);

  #countPlusOne = computed(() => this.#count.get() + 1);

  render() {
    return html`<p>One-indexed count: ${watch(this.#couplePlusOne)}`;
  }
}
```

Here I think `state()` and `computed()` start pulling their weight, given how cheap they are to vend. Again, the standard API is purposefully not made to be ergonomic because they assume frameworks will wrap it, and because the API hangs off a global instead of being exported from a module.

Reply:
> Again, the standard API is purposefully not made to be ergonomic because they assume frameworks will wrap it

This just doesn't make sense to me as anyone reading any documentation has to carefully connect the dots.

However, I think a better reason to wrap is that the proposal is in flux and we don't users to have to change their code if it changes.